frontend/src/App.tsx

```diff
@@ -6,6 +6,7 @@ import FAQPage from './pages/FAQPage';
 import ReviewPage from './pages/ReviewPage';
 import LandlordPage from './pages/LandlordPage';
 import ProfilePage from './pages/ProfilePage';
+import SavedApartmentsPage from './pages/SavedApartmentsPage';
```

Comment:
Please change to `SavedPage` (because we also have saved landlords and saved reviews in this page)